### PR TITLE
Theedge 2802

### PR DESCRIPTION
**Pull request by acosferreira**
# Description

Add support to RHEL-91

Fixes # ([THEEDGE-2802](https://issues.redhat.com/browse/THEEDGE-2802))

## Type of change

What is it?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update
- [ ] Refactor

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] I run `make pre-commit` to check fmt/vet/lint/test-no-fdo

### CODE REVIEW

File: config/repo_config.go [hunk 1]
```diff
@@ -39,6 +40,7 @@ var DistributionsPackages = map[string][]string{
 	distRHEL86: RHEL8X,
 	distRHEL87: RHEL8X,
 	distRHEL90: RHEL90,
+	distRHEL91: RHEL90,
```
Comment (pavankrao) [inline config/repo_config.go:43]: @acosferreira: should this be distRHEL91: RHEL9X similar to the v8s?

**Review by mgold1234 — APPROVED** [review]
/lgtm

File: config/repo_config.go (hunk 1 above)
Comment (acosferreira) [reply config/repo_config.go:43]: for while there is no changes on 90 and 91, we created x on 8 because the diff between 84,85 and 86,87

Comment (pavankrao) [reply config/repo_config.go:43]: sounds good @acosferreira , thanks!

**Review by pavankrao — APPROVED** [review]

**Pull request merged**